package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openshift-assisted-ui-lib",
-  "version": "0.0.0",
+  "version": "2.16.3",
```

Comment:
changing the version is a mistake, right ?

Reply:
Yes.